docs/_partner_api/changelog.md

```diff
@@ -7,6 +7,20 @@ redirect_from: /partner_api/
 
 # Changelog
 
+## 2021-07-28
+### Officially released `v1/profiles/me`
```

Comment:
Maybe we want to just say "Added `v1/profiles/me`"?  I'm unsure how widely known this endpoint was so it might be better to just err on the side of "oh this is brand new" 🤷 .  MG or Powell can advise here.

Reply:
@jonatascastro12 I agree. Let's just go with "added". Thanks all!